code/modules/mob/living/carbon/human/species_types/jellypeople.dm

```diff
@@ -572,16 +578,17 @@
 		return
 	CHECK_DNA_AND_SPECIES(H)
 
-	if(species.current_extract)
-		species.extract_cooldown = world.time + 100
```

Comment:
this line is actually important; it keeps you from spamming out do_after() extract activations